addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -18,6 +18,32 @@
         </xpath>
     </template>
 
+    <template id="product_cart_lines" inherit_id="website_sale.cart_lines" name="Wishlist Button">
+        <div class="o_wsale_remove_from_cart" position="inside">
+            <span class="d-none d-md-inline-block ms-2">
+                |
```

Comment:
I think we should go with a border or another CSS trick because a simple `|` char looks odd IMHO.

![image](https://github.com/user-attachments/assets/7eafb97d-56d9-4e42-b9fb-8d0637422bf4)

Reply:
Yes I agree, giving a border would definitely be better.